app/queries/servers/system.ts

```diff
@@ -174,6 +174,12 @@ export const observeLicense = (database: Database): Observable<ClientLicense | u
     );
 };
 
+export const observeLicenseBooleanValue = (database: Database, key: keyof ClientLicense) => {
+    return observeLicense(database).pipe(
+        switchMap((license) => of$(license?.[key] === 'true')),
+    );
```

Comment:
Add a `distinctUntilChanged` to this pipe, please.

Reply:
done.  Could you walk me through a case where this addition would be needed, or used?

Reply:
The license is one single object, so changes in the license will give a whole new object. If we don't use this, every time we have an update on the license (for example, websocket event) we would retrigger all pipes that come from the license. If we add this, we only trigger the pipes that have a value different.

License does not change so often, so probably is not such a big deal, but this is more important for config values.